proto/neutron/state_verifier/v1/genesis.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package neutron.state_verifier.v1;
+
+import "ibc/lightclients/tendermint/v1/tendermint.proto";
+
+option go_package = "github.com/neutron-org/neutron/v5/x/state-verifier/types";
+
+// ConsensusState describes a "light" consensus state of Neutron at a particular height
```

Comment:
please read the documentation policy and apply the guidelines https://www.notion.so/hadron/WIP-Neutron-team-documentation-policy-14485d6b9b10808e92b8e309fdf70d91

Reply:
I think it's fixed, please check